packages/editor/src/store/test/reducer.js

```diff
@@ -1826,7 +1826,12 @@ describe( 'state', () => {
 					} ],
 			} );
 
-			expect( state ).toBe( original );
```

Comment:
This change defeats the purpose of what the test was meant to verify (a change in reference). We might want both to verify that the selection changes to the last of the replacement blocks, and one where we confirm that a state reference wouldn't change if the selection is not changing (e.g. if original state's selected block was the "wings" block).

Reply:
Pushed 76c3aa23cc783b153a2b5d34714be0c1c79a.